src/styles/global.module.scss

```diff
@@ -25,7 +25,7 @@
 body,
 html {
   font-family: $font-family-1;
-  font-size: $font-size-normal;
+  font-size: 10px;
```

Comment:
Doesn't this change many of things ? why is it needed ?

Reply:
Because the timebar has `rem`s everywhere, so it will use the `font-size` defined in the body as "multiplier". In map-client, it was stupidly defined to something different than the browser defaults, so that's the only way I found to fix this, but am I overlooking something? (it has no impact because it seems everything in map client has an overridden font size anyways)